compiler/one-cmds/one-infer

```diff
@@ -206,6 +209,16 @@ def main():
     # run backend driver
     _utils._run(infer_cmd, err_prefix=ntpath.basename(driver_path))
 
+    # run post process script if it's given
+    if _utils._is_valid_attr(args, 'post_process'):
+        # Currently, it is only supported to use one-venv.
```

Comment:
what is `one-venv` ? if you meant virtual environment then I think this comment is needless as all the commands can only run in virtual environment.